pay-id-api-spec/pay-id.v1.yml

```diff
@@ -60,4 +97,59 @@ components:
       required:
         - addressDetailsType
         - addressDetails
+    Invoice:
+      title: Invoice
+      type: object
+      properties:
+        nonce:
+          type: string
+        expirationTime:
```

Comment:
Updated expiration time to `number` and using unix timestamp. (See my latest request in slack)

Reply:
talked offline, leaving this typed as a string.